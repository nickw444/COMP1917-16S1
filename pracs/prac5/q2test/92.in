92
2 -10 5 -10 3 4 -4 -10 0 2 5 5 -9 3 7 1 3 3 -7 6 -8 9 -1 -4 -6 -5 0 -4 -6 -7 4 4 -3 10 3 3 10 -4 -7 -9 6 9 0 7 10 7 -4 -10 -7 -4 8 5 1 -6 -7 6 10 8 10 1 -2 1 2 4 5 -4 -1 3 -1 -10 3 -6 4 5 -7 -3 -3 -4 4 -10 -3 -8 0 -5 0 3 3 -3 5 2 10 -2
