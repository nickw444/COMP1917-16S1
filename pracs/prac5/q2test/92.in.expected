average of array: 0.108696
